Mage.Sets/src/mage/cards/t/TribuneOfRot.java

```diff
@@ -0,0 +1,81 @@
+
+package mage.cards.t;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.AttacksTriggeredAbility;
+import mage.abilities.effects.OneShotEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Outcome;
+import mage.constants.SubType;
+import mage.constants.Zone;
+import mage.game.Game;
+import mage.game.permanent.token.Token;
+import mage.game.permanent.token.SaprolingToken;
+import mage.players.Player;
+
+import java.util.Objects;
+import java.util.UUID;
+
+/**
+ * @author BenGiese22
+ */
+public final class TribuneOfRot extends CardImpl {
+
+    public TribuneOfRot(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{2}{B/G}{B/G}");
+
+        this.subtype.add(SubType.ELF);
+        this.subtype.add(SubType.SHAMAN);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(3);
+
+        // Whenever Tribune of Rot attacks, mill two cards. For each creature card milled this way, createa  1/1 green Saproling craeture token.
+        this.addAbility(new AttacksTriggeredAbility(new TribuneOfRotEffect()));
+
+    }
+
+    private TribuneOfRot(final TribuneOfRot card) { super(card); }
+
+    @Override
+    public TribuneOfRot copy() { return new TribuneOfRot(this); }
+}
+
+class TribuneOfRotEffect extends OneShotEffect {
+
+    private static final Token token = new SaprolingToken();
+
+    TribuneOfRotEffect() {
+        super(Outcome.PutCreatureInPlay);
+        this.staticText = "For each creature card milled this way, create a 1/1 green Saproling creature token.";
+    }
+
+    private TribuneOfRotEffect(final TribuneOfRotEffect effect) { super(effect); }
+
+    @Override
+    public TribuneOfRotEffect copy() { return new TribuneOfRotEffect(this); }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        int numOfCreatureCardsMilled = 0;
+        Player player = game.getPlayer(source.getControllerId());
+        if (player != null) {
+            numOfCreatureCardsMilled += player
+                    .millCards(2, source, game)
+                    .getCards(game)
+                    .stream()
+                    .filter(Objects::nonNull)
+                    .filter(card -> game.getState().getZone(card.getId()) == Zone.GRAVEYARD)
+                    .filter(card1 -> card1.isCreature(game))
```

Comment:
this chain seems like it shouldn't be necessary... what's wrong with calling `count(StaticFilters.FILTER_CARD_CREATURE, game)` directly after `millCards()`?